fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts

```diff
@@ -216,10 +213,10 @@ export class FbwAircraftSentryClient {
       );
     }
 
-    const integrations: Integration[] = [new CaptureConsoleIntegration({ levels: ['error'] })];
+    const integrations = [Sentry.captureConsoleIntegration({ levels: ['error'] })];
 
     if (config.enableTracing) {
-      integrations.push(new BrowserTracing());
+      integrations.push(Sentry.browserTracingIntegration());
     }
```

Comment:
question: Just adding `browserTracingIntegration` doesn't enable Sentry's full tracing capabilities.
With just the integration, you get distributed tracing between services to connect errors in Sentry between individual services.

If you want to measure and record performance data, you also need to add `tracesSampleRate > 0` to your `Sentry.init` options. Let me know if you want to change this. Can also open a separate PR to keep things atomic :)

Reply:
It's been a while since I set this up - does the performance data measurement depend on React? We're in the process of rewriting the EFB to use an msfs-specific JS framework, so it may not be worth to keep this it if the integration only works with React or other web frameworks.

Reply:
> does the performance data measurement depend on React?

Since you're using `@sentry/browser` there's nothing framework-specific in the package. What you'll get from performance monitoring is essentially a span tree of how long e.g. browser resource loading takes, web vitals like LCP. More [information here](https://docs.sentry.io/product/performance/). To be honest, I don't know how well this translates to the MSFS renderer, so take what's in these docs with a grain of salt :)

If you want, we can give it a try but I suggest we open a separate PR to revert more easily if we need to

(there's also `@sentry/react` with some React-specific instrumentation but considering you're moving away from React I think it's better to stay with `@sentry/browser).